src/makeRobotAccountant.js

```diff
@@ -6,7 +6,25 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let calls = 0;
+
+  function getSum(x) {
+    return function(y) {
+      calls++;
+
+      if (calls < 4) {
+        return x + y;
+      }
+
+      if (calls > 4 && calls % 2 !== 0) {
+        return x + y;
+      } else {
+        return 'Bzzz... Error!';
+      }
```

Comment:
you can use ligical or insead block if  :  `if (calls > 4 && calls % 2 !== 0 || calls < 4)`

Reply:
better return an error if calls even and more than 3 if not return sum